roles/istio/templates/default-seldon-gateway.yaml.j2

```diff
@@ -23,6 +23,10 @@ spec:
       protocol: HTTPS
```

Comment:
@RafalSkolasinski can I "build" this locally somehow to ensure it compiles properly? Also maybe we need CI for this repo?

Reply:
There is no compilation involved, but there is a Makefiel target that you can use to install local copy of collections to test